### ISSUE TITLE
Error: undefined method 'address' for Socket::UNIXAddress (compile-time type is Socket::Address+)

### ISSUE THREAD

**Issue by bettinson** [luckyframework/lucky#1643]
**Describe the bug**
Type mismatch for `Socket::UNIXAddress`

**To Reproduce**
`ip_address = context.request.remote_address.try(&.address) || "N/A"`

from https://luckyframework.org/guides/http-and-routing/http-handlers#remoteiphandler

**Screenshots/code**
![image](https://user-images.githubusercontent.com/780179/148474220-c098d6a9-971f-47b0-affd-58e29f6c304b.png)

**Versions (please complete the following information):**
 - Lucky version 0.29.0
 - Crystal version Crystal 1.2.2 (2021-11-10)
 - OS: MacOS

**Comment by jwoertink** [comment]
Yeah, I feel like we should be able to update the handler to better handle this. You should have to do all the guessing and stuff with it

https://github.com/luckyframework/lucky/blob/08a70e191b6a242cdf972750219dc732e2d5ec16/src/lucky/remote_ip_handler.cr#L10-L32
